scripts/get_container_images.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+declare ARCHIVE_DESTINATION
+declare -r LONG_OPTS='destination:'
+declare -r SHORT_OPTS='d:'
+
+print_error() {
+    echo -e "\e[91m$*\e[39m" >&2
+}
+
+print_ok() {
+    echo -e "\e[92mOK\e[39m"
+}
+
+print_ko() {
+    print_error "KO"
+}
+
+execute_command() {
+    echo -n "$1... "
+    shift
+    if ! error=$("$@" 2>&1); then
+        print_ko
+        print_error "Failing command: $*\nError: $error"
+        exit 1
+    fi
+    print_ok
+}
+
+pull_container_images() {
+    for container_image; do
+        execute_command \
+            "Pulling container image $container_image" \
+            docker pull "$container_image"
+    done
+}
+
+save_container_images() {
+    local image_archive container_image_without_host
+
+    for container_image; do
+        container_image_without_host=${container_image##*/}
+        image_archive=${container_image_without_host%@*}.tar
+        execute_command \
+            "Saving container image $container_image to $image_archive" \
+            docker save "$container_image" \
+                --output "$ARCHIVE_DESTINATION/$image_archive"
```

Comment:
Cant we compress the image ? ` | gzip -c > XX` , this will decrease the iso size and we can decompress in the load time

Reply:
Can we do better than `gz`? Should check whether `bz2` or `xz` are available on a bare-bones CentOS7 by default.

Reply:
FYI, we have issue with bz with S3 archive because, for a very minimal gain, we take several more minutes just to uncompress it. @alexandre-allard-scality will now better about it, but I know he wanted to switch back to gzip for the docker images and the whole archive.

Reply:
Yeah. I experimented with both `bzip2` and `xz` last night. At least for compression, `bzip2` has a small gain compared to `gzip`, the gain from `xz` is more impactful (30+%). However, compression time with `xz` is a multiple (for some big factor) of the `gzip` compression time.

Maybe we should have some kind of 'devel' builds which use `gzip`, and `release` builds using `xz` one day.

Reply:
I've done some tests on gz, bz2 & xz for Federation stuff and it seems that gz is the best time to uncompress / size ratio.
But, if we don't care about the time used to uncompress, we should use xz which has the best compression in term of size.
That said, TS were asking for something faster, in general the overhead in the size of the archive is not really a problem, but the uncompression time can be painful.